src/test/java/com/amplitude/api/BaseTest.java

```diff
@@ -87,8 +99,18 @@ public void setUp() throws Exception {
      * calling this method or passing false for withServer.
      */
     public void setUp(boolean withServer) throws Exception {
-        ShadowApplication.getInstance().setPackageName("com.amplitude.test");
-        context = ShadowApplication.getInstance().getApplicationContext();
+        context = ApplicationProvider.getApplicationContext();
+        packageManager = RuntimeEnvironment.application.getPackageManager();
+        shadowPackageManager = shadowOf(packageManager);
+
+        ApplicationInfo applicationInfo = new ApplicationInfo();
+        applicationInfo.packageName = TEST_PACKAGE_NAME;
+
+        PackageInfo packageInfo = new PackageInfo();
+        packageInfo.packageName = TEST_PACKAGE_NAME;
+        packageInfo.versionName = TEST_VERSION_NAME;
```

Comment:
👍  this probably fixed the version failure, so the database test returned 3 errors instead of 4